packages/cards-editor-middleware/src/index.ts

```diff
@@ -13,6 +13,32 @@ export const enum ApiRoutes {
     cardsStore = '/cards/store',
     i18nStore = '/editor/i18n'
 }
+export const DEFAULT_THEME = 'sap_horizon';
+
+const UI5_LIBS = [
```

Comment:
@heimwege  We have more UI5 libraries than added here. Should we add them all here or there is already some generic way to get existing libraries. It's rare but number of libraries can change with UI5 version.

Libraries with  SAP UI5 1.130: https://sapui5.hana.ondemand.com/#versioninfo.html

Reply:
@tobiasqueck the list is a copy from preview-middleware (see [here](https://github.com/SAP/open-ux-tools/blob/main/packages/preview-middleware/src/base/config.ts#L105)). Is that list outdated and needs to be adjusted?

Reply:
I think this was originally done by @devinea for the load-reuse-libs script. Maybe he can shed some light about static vs. dynamically loading it. Especially now, since we preload this file anyhow to detect 2.0, right?

Reply:
https://sonarcloud.io/code?id=SAP_open-ux-tools&pullRequest=2613
Sonar duplication is also because of these 15 lines:
```
function getUI5Libs(manifest: Partial<Manifest>): string {
    const libs = manifest['sap.ui5']?.dependencies?.libs ?? {};
    // add libs that should always be preloaded
    libs['sap.m'] = {};
    libs['sap.ui.core'] = {};
    libs['sap.ushell'] = {};
    return Object.keys(libs)
        .filter((key) => {
            return UI5_LIBS.some((substring) => {
                return key === substring || key.startsWith(substring + '.');
            });
        })
        .join(',');
}
```


I think it makes sense to maintain this function in a single place & import this ? That way we will not have the duplication of these 15 lines which is shown here https://sonarcloud.io/code?id=SAP_open-ux-tools&pullRequest=2613 ?? @tobiasqueck @heimwege @devinea

Reply:
> How about reading the list from [sap-ui-version.json](https://ui5.sap.com/resources/sap-ui-version.json)? .themes['base'].libraries seems to be an interesting list

@heimwege Yes this looks like a good replacement to the hardcoded `UI5_LIBS`.

Reply:
I just double-checked and found out that we can't use the `themes` property because it does not exist in case you don't use the proxy middleware but serve the ui5 sources directly via ui5 tooling
![image](https://github.com/user-attachments/assets/5195a777-b8d3-40dc-9f91-bd99c7e3bc81)

Reply:
Let's fix this in preview-middleware first. Opened https://github.com/SAP/open-ux-tools/issues/2683